Userland/Libraries/LibGfx/ImageFormats/PNGLoader.cpp

```diff
@@ -1209,9 +1208,10 @@ static bool process_chunk(Streamer& streamer, PNGLoadingContext& context)
         dbgln_if(PNG_DEBUG, "Bail at chunk_size");
         return false;
     }
-    u8 chunk_type[5];
-    chunk_type[4] = '\0';
-    if (!streamer.read_bytes(chunk_type, 4)) {
+
+    Array<u8, 4> chunk_type_buffer;
+    StringView const chunk_type { chunk_type_buffer.span() };
+    if (!streamer.read_bytes(chunk_type_buffer.data(), 4)) {
```

Comment:
s/4/chunk_type_buffer.size()/

Reply:
Shouldn't be a concern for too long :eyes: